examples/text-generation/README.md

```diff
@@ -361,22 +357,6 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
 --bf16 \
 --reuse_cache \
 --trim_logits \
---fp8
-```
-
-Here is an example to measure the tensor quantization statistics on phi-2 with 1 card:
-
-```bash
-QUANT_CONFIG=./quantization_config/maxabs_measure.json python run_lm_eval.py \
--o acc_phi-2_bs1_measure.txt  \
---model_name_or_path microsoft/phi-2 \
---use_hpu_graphs \
---use_kv_cache \
---max_new_tokens 100 \
---batch_size 1 \
---trim_logits \
---reuse_cache \
---bf16
```

Comment:
Why removing the measure command for phi but not for other models?

Reply:
It's not removed, I put it back in the merge commit afterwards

Reply:
I still see it removed: https://github.com/huggingface/optimum-habana/pull/962/files#diff-d8103ff4973ad399e0cc6fc01efe091f716c7b91510ab34a05ea1d70f180459bL367

Reply:
@regisss fixed it now